cityofaustin/atd-moped
# Adds "Contractors" and "Contract #" to project search + advanced filters

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10047

## Testing
**URL to test:** 
https://10047_add_contracts_project_search_filters--atd-moped-main.netlify.app/moped/

**Steps to test:**
go to a project, select the funding tab, and add two or more contracts with different contractor names and numbers
go to the project list view, scroll all the way to the right and you should see the names and numbers listed with commas
go to the project list view advanced, and search the 'contractors' and 'contract numbers' fields with the values you entered in the funding tab, results should appear

---
#### Ship list
